### PR TITLE
chore(eslint): apply eslint and prettier company standard

### PR DESCRIPTION
**Pull request by PeterMK85**
### Description

Applied `autofix`, and maximized the number of warnings.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "test:coverage": "env NODE_ENV=test TZ=UTC jest --coverage",
     "test:ci": "env NODE_ENV=test TZ=UTC jest --bail --coverage --maxWorkers=4 --ci",
     "lint:tsc": "tsc -p ./tsconfig.json",
-    "lint": "eslint src/ --ext .js,.jsx,.ts,.tsx --color --fix",
+    "lint:ci": "davinci-syntax lint code . --max-warnings 264 --quiet",
```
Comment (PeterMK85) [inline package.json]: maximized the current warning amount

File: package.json [hunk 2]
```diff
@@ -90,14 +91,12 @@
     "@storybook/storybook-deployer": "^2.8.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.4.0",
+    "@toptal/davinci": "^0.1.1",
     "@types/classnames": "^2.2.9",
     "@types/jest": "^24.0.23",
     "@types/react-dom": "^16.9.4",
-    "@typescript-eslint/eslint-plugin": "^2.6.0",
```
Comment (PeterMK85) [inline package.json]: removed unused packages

File: .eslintrc.js [hunk 3]
```diff
@@ -1,155 +1,24 @@
...
-      'error',
-      {
-        checkContextTypes: true
-      }
-    ],
-    'object-curly-spacing': ['error', 'always'],
-    'newline-after-var': ['error', 'always'],
-    'import/order': [
-      'error',
+    'react/jsx-props-no-spreading': 'warn',
```
Comment (PeterMK85) [inline .eslintrc.js]: no manual adjustments, everything which is currently an error, transformed to warning, everything else fixed by autofix

File: .eslintrc.js [hunk 4]
```diff
@@ -1,155 +1,24 @@
...
-        assignment: 'parens-new-line',
-        return: 'parens-new-line',
-        arrow: 'parens-new-line',
-        condition: 'parens-new-line',
-        logical: 'parens-new-line'
-      }
-    ],
-    'react/jsx-max-depth': 'off',
-    'react/jsx-filename-extension': [
+    '@typescript-eslint/no-empty-function': 'warn',
```
Comment (havenchyk) [inline .eslintrc.js]: This should be fixed soon in davinci

**Pull request merged**